### PR TITLE
Adding starRating()

### PR DESCRIPTION
**Pull request by mlkirby73**
Added the starRating() method to Review.java. @ncannapruett @JnPruett

### CODE REVIEW

**Comment by ncannapruett** [comment]
I tried your starRating() method into my code, and it imputed the wrong output. Why did you choose to make the if statement:
if (sentiment < -3)	
{	
       rating = 1;	
}
instead of: 
if(sentiment < 0) 
{
       return 1; 
}

File: Review.java [hunk 1]
```diff
@@ -165,4 +165,31 @@ public static String randomAdjective()
...
     }
   }
+  public static int starRating(String fileName)
+   {
+     double sentiment = totalSentiment(fileName);
+     int rating;
+     
+     if (sentiment < -3)
+     {
+       rating = 1;
```
Comment (JnPruett) [inline Review.java:176]: You can return an int without creating a variable but otherwise looks good to merge!

**Comment by mlkirby73** [comment]
> I tried your starRating() method into my code, and it imputed the wrong output. Why did you choose to make the if statement:
> if (sentiment < -3)
> {
> rating = 1;
> }
> instead of:
> if(sentiment < 0)
> {
> return 1;
> }

**Comment by mlkirby73** [comment]
@ncannapruett starting at -3 is just the way I did the scaling. I did it in increments of 3 and it compiled fine on my computer.

File: Review.java (hunk 1 above)
Comment (mlkirby73) [reply Review.java:176]: Thanks for the feedback!

**Pull request merged**